src/display/Display.spec.js

```diff
@@ -32,4 +33,32 @@ test('Displays Open if closed prop is false', () => {
     const open = getByText(/open/i);
 
     expect(open.textContent).toMatch('Open');
-})
\ No newline at end of file
+})
+
+test('Displays Locked if the locked prop is true', () => {
+    const {getByText} = render(<Display locked={true} />);
+    const locked = getByText(/locked/i);
+
+    expect(locked.textContent).toMatch('Locked');
+})
+
+test('Displays Unlocked if the locked prop is false', () => {
+    const {getByText} = render(<Display locked={false} />);
+    const unlocked = getByText(/unlocked/i);
+
+    expect(unlocked.textContent).toMatch('Unlocked');
+})
+
+test('When using locked/closed the red-led class is actived', () => {
+    const {getByText} = render(<Display locked={true} closed={true}/>)
+    // const lockedAndClosed = getByText(/closed/i);
+    
+    expect(getByText(/closed/i)).toHaveClass('red-led');
+})
+
```

Comment:
You could validate that Both closed and Locked have the red-led as well as closed and unlocked to have read and then green - led we  like below
```
const { getByText } = render(<Display closed={true} locked={true} />);
    const closed = getByText(/closed/i);
    const locked = getByText(/locked/i);
    expect(closed).toHaveClass("red-led");
    expect(locked).toHaveClass("red-led");
```
Just remember if what state the button takes sometimes might dictate what to test. If you can have multiple states of the button then you may want to test those multiples states within reason.